src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -16,4 +16,9 @@ public CommandResult execute() {
         addressBook.clear();
         return new CommandResult(MESSAGE_SUCCESS);
     }
+
+    @Override
+    public boolean isMutating(){
```

Comment:
Remember to update your overriden method `isMutating()` with header comments as its behavior is different from the parent class implementation.